src/models/campaign.ts

```diff
@@ -151,9 +153,9 @@ export default class Campaign {
             .keyBy("code")
             .mapValues("dataElements")
             .value();
-
-        if (!vaccinationAttribute) {
-            return { status: false, error: "Metadata not found: vaccinationAttributeId" }
+        if (!dashboardId) return { status: false, error: 'Error creating dashboard' };
```

Comment:
Does this merit an early return or should I also add it to the if/else chain?

Reply:
The if/else chain is already there, so inside the chain.

Note that the chain can be flattened (only one indentation level) by using if/else-if/else-if/..../else, for example here:

```
} else {
            if (!categoryComboTeams) {
                return {status: false, error: `Metadata not found: teamsCode=${teamsCode}`};
} else {
```

Reply:
done